Cargo.lock

```diff
@@ -648,6 +648,16 @@ dependencies = [
  "subtle",
 ]
 
+[[package]]
```

Comment:
Curious, I thought the action task is only for non-breaking changes. How come there are new dependencies being added?

Reply:
This means a new version of some package has an added dependency which gets pulled here as well by the resolver.  Cargo update follows semver conventions so as long as the pulled crates doesn't tag the change with the new dependency as a breaking change, it will show up here.

Reply:
Sounds good, thanks.